board/arm-64bit-toolchain.cmake

```diff
@@ -0,0 +1,52 @@
+set(CMAKE_SYSTEM_NAME Linux)
```

Comment:
It could be worth adding some documentation to the top of this file, since users may need to modify it for their particular system.